kia-lib/src/test/kotlin/com/faforever/ice/telemetry/TelemetryClientTest.kt

```diff
@@ -52,9 +54,8 @@ class TelemetryClientTest {
 
     @Test
     fun `test init connects and registers as peer`() {
-        verify {
-            anyConstructed<TelemetryClient.TelemetryWebsocketClient>().connect()
-
+        verify(timeout = 1000) {
```

Comment:
With the sendloop running on a separate thread testing is weird since we need to wait for the sendloop thread to process the message before verifying. I'm not sure if this is the best way, but my understanding is the timeout on verify doesn't wait the full time if the conditions complete early.

Reply:
From my experience the timeout doesn't work at all. Use Awaitility await()
Example: https://github.com/FAForever/kotlin-ice-adapter/blob/f3fcf74a87ff6d5ab5f4227c72390ede41d499d8/kia-lib/src/test/kotlin/com/faforever/ice/peering/UdpSocketBridgeTest.kt#L45